packages/wallet-sdk/src/relay/MobileRelay.ts

```diff
@@ -25,9 +25,7 @@ export class MobileRelay extends WalletLinkRelay {
     return {
       promise: new Promise(() => {
         const location = getLocation();
-        location.href = `https://www.coinbase.com/connect-dapp?uri=${encodeURIComponent(
-          location.href
-        )}`;
+        location.href = `https://go.cb-w.com/dapp?cb_url=${encodeURIComponent(location.href)}`;
```

Comment:
reverting 
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/d4ab15ab-b12b-4b7d-8eef-0852cffee863)

`dapp` is also a correct path

Reply:
Do we need any walletlink changes?

Reply:
no. this is handled by branch.io